src/main/java/uk/gov/hmcts/divorce/solicitor/client/organisation/OrganisationUser.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.divorce.solicitor.client.organisation;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OrganisationUser {
+    private String userIdentifier;
```

Comment:
this seems fine but weird how all an OrganisationUser has is one field.

Reply:
It has more but we're just ignoring the rest during deserialisation